lib/qx/tool/compiler/app/Translation.js

```diff
@@ -187,7 +187,7 @@ module.exports = qx.Class.define("qx.tool.compiler.app.Translation", {
                         entry.comments.reference = {};
                       }
                       const ref = entry.comments.reference;
-                      comment.match(/[\w/\.]+:\d+/g).forEach(entry => {
+                      (comment.match(/[\w/\.]+:\d+/g) || []).forEach(entry => {
```

Comment:
Looking at the code starting with line 169, it seems that `comment`, too, can be null if the line being matched contains only `#`. Maybe this should, in fact, check for comment being non-null?
```
(comment && comment.match(/[\w/\.]+:\d+/g) || []).forEach(entry => {
```

Reply:
@johnspackman Can you confirm?

Reply:
that's a good point; as they're comments, there should be no implicit assumption about formatting (quite apart from the fact that an exception causes an inexplicable failure)